tensorboard/plugins/profile/pod_viewer/topology_graph/topology-graph.ts

```diff
@@ -411,33 +359,47 @@ Polymer({
         .on('mouseout', function() {
             d3.select(this).classed('cell-hover', false).style('opacity', 1.0);
             d3.select(parent.$.tooltip).classed('hidden', true);
-        });
+        })
+        .transition()
+        .duration(TRANSITION_DURATION);
     cards.exit().remove();
   },
   /**
    * Draw the UI of chip to chip links.
    */
-  drawLinks: function(linkData: Array<podviewer.proto.ChannelInfo>) {
-    if (!linkData || linkData.length == 0 || !this._gLink) {
+  drawLinks: function(linkData: Array<Array<number>>) {
```

Comment:
`Array<Array<number>>` looks unwieldy. Would it be possible to define a type for the tuple?
```ts
// Option 1:
type SrcDestIds = [number, number];
drawLinks(linkData: Array<SrcDestIds>) {
  ...
}

// Option 2:
type SrcDestIds = {srcId: number, destId: number};
drawLinks(linkData: Array<SrcDestIds>) {
  ...
}
```